manifest/manifest_test.go

```diff
@@ -1,170 +1,159 @@
-	require.NoError(t, err)
-
-	mani, err := sdl.Manifest()
-	require.NoError(t, err)
-
-	_, kmgr := testutil.NewNamedKey(t)
-	signer := testutil.Signer(t, kmgr)
-
-	deployment := testutil.DeploymentAddress(t)
-
-	mr, _, err := SignManifest(mani, signer, deployment)
-	assert.NoError(t, err)
-
-	_, otherKmgr := testutil.NewNamedKey(t)
-	otherSigner := testutil.Signer(t, otherKmgr)
-	otherMr, _, err := SignManifest(mani, otherSigner, deployment)
-	assert.NoError(t, err)
-
-	mr.Key = otherMr.Key
-
-	_, err = verifySignature(mr)
-	assert.Error(t, err)
-}
-
-func TestVerifyDeploymentTenant(t *testing.T) {
-	info, kmgr := testutil.NewNamedKey(t)
-	signer := testutil.Signer(t, kmgr)
-	tenant := info.GetPubKey().Address().Bytes()
-	deployment := testutil.Deployment(tenant, 1)
-	mani := &types.Manifest{}
-	_, _, err := SignManifest(mani, signer, deployment.Address)
-	require.NoError(t, err)
-	err = verifyDeploymentTenant(deployment, info.GetPubKey().Address().Bytes())
-	assert.NoError(t, err)
-}
-
-func TestVerifyDeploymentTenant_InvalidKey(t *testing.T) {
-	info, kmgr := testutil.NewNamedKey(t)
-	signer := testutil.Signer(t, kmgr)
-	tenant := info.GetPubKey().Address().Bytes()
-	deployment := testutil.Deployment(tenant, 1)
-	mani := &types.Manifest{}
-	_, _, err := SignManifest(mani, signer, deployment.Address)
-	require.NoError(t, err)
-	err = verifyDeploymentTenant(deployment, info.GetPubKey().Address().Bytes())
-	assert.NoError(t, err)
-}
-
-func TestVerifyRequest(t *testing.T) {
-	info, kmgr := testutil.NewNamedKey(t)
-	signer := testutil.Signer(t, kmgr)
-	tenant := info.GetPubKey().Address().Bytes()
-	mani := &types.Manifest{}
-	version, err := Hash(mani)
-	require.NoError(t, err)
-	deployment := testutil.Deployment(tenant, 1, version)
-	mreq, _, err := SignManifest(mani, signer, deployment.Address)
-	require.NoError(t, err)
-	err = VerifyRequest(mreq, deployment)
-	assert.NoError(t, err)
-}
-
-func TestHash(t *testing.T) {
-	sdl, err := sdl.ReadFile("../_docs/deployment.yml")
-	require.NoError(t, err)
-
-	mani, err := sdl.Manifest()
-	require.NoError(t, err)
-
-	_, err = Hash(mani)
-	assert.NoError(t, err)
-}
-
-func TestVerifyHash(t *testing.T) {
-	sdl, err := sdl.ReadFile("../_docs/deployment.yml")
-	require.NoError(t, err)
-
-	mani, err := sdl.Manifest()
-	require.NoError(t, err)
-
-	hash, err := Hash(mani)
-	require.NoError(t, err)
-
-	otherHash, err := Hash(mani)
-	require.NoError(t, err)
-
-	assert.Equal(t, hash, otherHash)
-}
-
-func TestVerifyHash_Invalid(t *testing.T) {
-	sdl, err := sdl.ReadFile("../_docs/deployment.yml")
-	require.NoError(t, err)
-
-	mani, err := sdl.Manifest()
-	require.NoError(t, err)
-
-	hash, err := Hash(mani)
-	require.NoError(t, err)
-
-	otherHash, err := Hash(&types.Manifest{
-		Groups: []*types.ManifestGroup{
-			{
-				Name: "otherManifest",
-			},
-		},
-	})
-	require.NoError(t, err)
-
-	assert.NotEqual(t, hash, otherHash)
-}
+// func TestSignManifest(t *testing.T) {
```

Comment:
Is there a chance to use these in future?

Reply:
Code related to these tests is commented, so I just kept them commented